Makefile

```diff
@@ -0,0 +1,55 @@
+WORKSPACE ?= VelocidiSDK.xcworkspace
+TEST_SDK ?= 14.4
+TEST_DEVICE ?= iPhone 11 Pro Max
+
+XCARGS := -workspace $(WORKSPACE) \
+					-sdk "iphonesimulator$(TEST_SDK)" \
+					-destination "platform=iOS Simulator,OS=$(TEST_SDK),name=$(TEST_DEVICE)" \
+					GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+
+all: clean build
+
+# we have to clean schemas independently because xcode does not allow to clean all schemes in a workspace
+clean:
+	xcodebuild $(XCARGS) -scheme VelocidiSDK clean | xcpretty && \
+	xcodebuild $(XCARGS) -scheme ObjcExample clean | xcpretty && \
+	xcodebuild $(XCARGS) -scheme SwiftExample clean | xcpretty
+
+build:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK build | xcpretty
+
+test:
```

Comment:
Shouldn't the `test` target depend on the `build` target?

Reply:
Yes. Fixed in 173f363